### ISSUE TITLE
Starter project

### ISSUE THREAD

**Issue by petivagyok16** [ngrx/platform#526]
## I have a question about a starter project

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
</code></pre>

## What is the current behavior?

Currently, example app is wrapped inside the platform ngrx project, but i'd like to create a starter app for myself.
## Expected behavior:

i'd like to create a starter project with npm and angular cli based on the example-app which can be found in the platform project.
Is it recommended to use ngrx 4 without yarn, or not? Has yarn any advantage over npm (i'm not familiar with yarn) or is it totally okay if i cut out the parts i'm not interested in?
Basically, i'd clone the example-app as the main entry point, without the platform project wrapper. Packages would be installed via npm.

**Comment by brandonroberts** [comment]
You'd have an easier time creating a new CLI project from scratch and copying the contents of the example app folder into it. There are ngrx starter projects out there already you can reference also.
